cypress/components/confirm/confirm.cy.tsx

```diff
@@ -196,12 +200,12 @@ context("Testing Confirm component", () => {
     });
 
     it("should check Esc key is disabled", () => {
+      const callback = cy.stub();
```

Comment:
was this callback assigned anywhere?

Reply:
Callback is also appear for the event tests, but for this case, this callback is only assigned for this particular test